intel_extension_for_transformers/transformers/modeling/modeling_auto.py

```diff
@@ -137,6 +137,16 @@ def from_pretrained(cls, pretrained_model_name_or_path, *model_args, **kwargs):
                 except:
                     logger.error("Saved low bit model loading failed, please check your model.")
                     exit(0)
+        if kwargs.get("load_int8_model", False):
+            from intel_extension_for_transformers.llm.evaluation.models import TSModelCausalLMForITREX

```

Comment:
why we need a special class xxForITREX?

Reply:
because the model used ipex.optimize_transformers inference is different with optimum-intel `TSModelForCausalLM`, the main difference is the past_key_values format. So I inherited `TSModelForCausalLM` and overridden the forward function.

Reply:
then the class name should change to "ForIPEX"?